src/v2/Apps/Show/__tests__/ShowArtworks.jest.tsx

```diff
@@ -0,0 +1,194 @@
+import { MockBoot, renderRelayTree } from "v2/DevTools"
+import React from "react"
+import { ShowArtworksRefetchContainer } from "../components/ShowArtworks"
+import { graphql } from "react-relay"
+import { ShowArtworks_QueryRawResponse } from "v2/__generated__/ShowArtworks_Query.graphql"
+
+jest.unmock("react-relay")
+jest.mock("v2/Artsy/Router/useRouter", () => ({
+  useRouter: () => ({
+    match: {
+      location: { query: {} },
+    },
+  }),
+}))
+
+describe("ShowArtworks", () => {
+  const getWrapper = async (
+    response: ShowArtworks_QueryRawResponse = SHOW_ARTWORKS_FIXTURE
+  ) => {
+    return renderRelayTree({
+      Component: ({ show }) => {
+        return (
+          <MockBoot>
+            <ShowArtworksRefetchContainer show={show} />
+          </MockBoot>
+        )
+      },
+      query: graphql`
+        query ShowArtworks_Query($slug: String!) @raw_response_type {
+          show(id: $slug) {
+            ...ShowArtworks_show
+          }
+        }
+      `,
+      variables: { slug: "miart-2020" },
+      mockData: response,
+    })
+  }
+
+  it("renders correctly", async () => {
+    const wrapper = await getWrapper()
+    expect(wrapper.find("ArtworkFilterArtworkGrid").length).toBe(1)
+    expect(wrapper.find("GridItem__ArtworkGridItem").length).toBe(2)
+  })
+})
+
+const SHOW_ARTWORKS_FIXTURE: ShowArtworks_QueryRawResponse = {
```

Comment:
@mzikherman - new test code should take advantage of the MockPayloadGenerator -- we don't need any of this fixture stuff anymore. 

Check out this PR to see how things can be simplified all around: https://github.com/artsy/force/pull/6414

Reply:
oooo yes good call! Thank you. We should be using that for new tests.